File: src/main/java/pl/epoint/hackyeah/repository/search/UserSearchRepository.java
package pl.epoint.hackyeah.repository.search;

import pl.epoint.hackyeah.domain.User;
import org.springframework.data.elasticsearch.repository.ElasticsearchRepository;

/**
 * Spring Data Elasticsearch repository for the User entity.
 */
public interface UserSearchRepository extends ElasticsearchRepository<User, Long> {
}

File: src/main/kotlin/pl/epoint/hackyeah/repository/search/UserSearchRepository.kt
package pl.epoint.hackyeah.repository.search

import pl.epoint.hackyeah.domain.Player
import org.springframework.data.elasticsearch.repository.ElasticsearchRepository

/**
 * Spring Data Elasticsearch repository for the User entity.
 */
interface UserSearchRepository : ElasticsearchRepository<Player, Long>
